renovate.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>netlify/renovate-config:esm"],
+
+  "ignorePresets": [":prHourlyLimit2"],
+  "semanticCommits": "enabled",
+  "dependencyDashboard": true,
+
+  "packageRules": [
+    {
+      "matchFiles": ["package.json", "site/package.json"],
```

Comment:
should we include the testing fixtures here as well?

Reply:
I wasn't sure if there we might use ranges on purpose, but probably not.

Reply:
Done, it is now active for all npm dependencies